frontend/src/api/gasStation.ts

```diff
@@ -0,0 +1,49 @@
+import { AccountAuthenticator, AnyRawTransaction } from "@aptos-labs/ts-sdk";
+import axios from "axios";
```

Comment:
Can we just use the normal fetch API? I'd rather not bring in axios if we don't have to.

Reply:
Will remove